CHANGELOG.md

```diff
@@ -409,10 +423,3 @@ To dig deeper please feel free to follow the links mentioned below:
 - Fixed image location referencing an incorrect branch in the generated VSCode marketplace README (VSCODE-281, #333)
 
 
-## [v0.6.9](https://github.com/mongodb-js/vscode/releases/tag/v0.6.9) - 2021-07-27
```

Comment:
Huh... wonder why that has changed 🤔

Reply:
Hm... looks like this is an issue with response pagination. I'll take a stab at fixing the workflow.